src/server/kiwix-serve.cpp

```diff
@@ -324,17 +324,15 @@ int main(int argc, char** argv)
   auto libraryFileTimestamp = newestFileTimestamp(libraryPaths);
   auto curLibraryFileTimestamp = libraryFileTimestamp;
 
-  /* Infer ipMode from address */
-  kiwix::IpMode ipMode = kiwix::IpMode::ipv4;
-
-  if (address == "all"){
-    address = "";
-    ipMode = kiwix::IpMode::all;
-  } else if (address == "ipv4"){
-    address = "";
-  } else if (address == "ipv6"){
-    address = "";
-    ipMode = kiwix::IpMode::ipv6;
+  if (address == "all") {
+    address.clear();
+    ipMode = kiwix::IpMode::ALL;
+  } else if (address == "ipv4") {
+    address.clear();
+    ipMode = kiwix::IpMode::IPV4;
+  } else if (address == "ipv6") {
+    address.clear();
+    ipMode = kiwix::IpMode::IPV6;
```

Comment:
1. You lost the declaration of `ipMode`.
2. Please minimize the diff so that that it's obvious that this change is merely about the capitalization of the `IpMode` enum members.

Reply:
> 2. Please minimize the diff so that that it's obvious that this change is merely about the capitalization of the IpMode enum members.

... and the default IP mode becoming `AUTO`.